packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -536,7 +550,7 @@ class PathNode implements UpPath {
      * (like the field in the map, and possibly this entire PathNode and its parents if they are no longer needed.)
      */
     public afterEmptyField(key: FieldKey): void {
-        assert(this.status === Status.Alive, 0x40f /* PathNode must be alive */);
+        assert(this.status !== Status.Disposed, "PathNode must not be disposed");
```

Comment:
the documentation on Status.Dead says its invalid to access a node with that status. It seems like it should count as accessing.

Since Status is an internal type, it seems fair to assume that applies to internal usages like this.

Thus I think this should be put back how it was, or the documentation on Status.Dead and/or this function should be changed to allow this.

Reply:
Agreed. As described in the other comments, I would favor updating the documentation on Status.Dead (and renaming it).

Reply:
As described in the other comments, I've come around to the above proposal.